### DISCUSSION TITLE
How to apply Label Setup settings to multiple selected point layers in the legend ?

### DISCUSSION THREAD

**Discussion by abt5555** [DotSpatial/DotSpatial#1524] [Q&A] [answered]
Hi,

I wonder if it is possible to apply layer right click operations to all selected point layers in the legend? 

For example,

I would like select multiple point layers in the legend then mouse right click > Labeling > Label Setup and making the label settings and clicking apply ok. Is it possible to apply these settings to all selected point layers?

OR

Is it possible to select multiple layers in the legend then mouse right click > Remove Layer or Data > Export Data

Thank you,

**Comment by jany-tenaj** [comment, marked as answer]
Currently that is not possible.

**Comment by abt5555** [comment]
Thank you for the response
